homeassistant/components/zwave_js/api.py

```diff
@@ -1001,6 +1008,44 @@ async def websocket_provision_smart_start_node(
             "QR code version S2 is not supported for this command",
         )
         return
+
+    # Create an empty device if device_name is provided
+    if device_name := msg.get(DEVICE_NAME):
+        dev_reg = dr.async_get(hass)
+
+        # Create a unique device identifier using the DSK
+        device_identifier = (DOMAIN, f"provision_{provisioning_info.dsk}")
+
+        manufacturer = None
+        model = None
+
+        device_info = await driver.config_manager.lookup_device(
+            provisioning_info.manufacturer_id,
+            provisioning_info.product_type,
+            provisioning_info.product_id,
+        )
+        if device_info:
+            manufacturer = device_info.manufacturer
+            model = device_info.label
+
+        # Create an empty device
+        device = dev_reg.async_get_or_create(
+            config_entry_id=entry.entry_id,
+            identifiers={device_identifier},
+            name=device_name,
+            manufacturer=manufacturer,
+            model=model,
+        )
```

Comment:
I think we should be able to set `via_device_id` already here, since there will always be a controller node device known at this point.